sympy/vector/deloperator.py

```diff
@@ -13,19 +9,11 @@ class Del(Basic):
     mathematical expressions as the 'nabla' symbol.
     """
 
-    def __new__(cls, system):
-        if not isinstance(system, CoordSysCartesian):
-            raise TypeError("system should be a CoordSysCartesian")
-        obj = super(Del, cls).__new__(cls, system)
-        obj._x, obj._y, obj._z = system.x, system.y, system.z
-        obj._i, obj._j, obj._k = system.i, system.j, system.k
-        obj._system = system
-        obj._name = system.__str__() + ".delop"
-        return obj
+    def __new__(cls,):
```

Comment:
Compatibility break, we cannot just break the API by removing an argument (there are probably thousands of people having scripts using this code, we need a SymPy deprecation warning if we're gonna remove an argument).